libsel4cp/include/sel4cp.h

```diff
@@ -7,26 +7,30 @@
 
 #pragma once
 
+#include <stdbool.h>
 #include <stdint.h>
 #include <stdbool.h>
 
 #define __thread
 #include <sel4/sel4.h>
 
 typedef unsigned int sel4cp_channel;
+typedef unsigned int sel4cp_pd;
 typedef seL4_MessageInfo_t sel4cp_msginfo;
 
 #define MONITOR_EP 5
 #define BASE_OUTPUT_NOTIFICATION_CAP 10
 #define BASE_ENDPOINT_CAP 74
 #define BASE_IRQ_CAP 138
+#define BASE_TCB_CAP 202
```

Comment:
can we have the tool inject these defines somehow instead of hard coding

Reply:
I don’t see any benefit of doing so given they are constants. In addition the verification of the library depends on these being known rather than patched into the library after the fact.